_posts/blog/2016-12-19-running-on-fedora-25.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Running Accumulo on Fedora 25"
+author: Christopher Tubbs and Mike Miller
+reviewers: 
+---
+
+Apache Accumulo has been available in [Fedora] for the last few releases.
+Recently, the [maintainers] have updated it to version `1.6.6` and have made
```

Comment:
Could possibly replace `it` with `Accumulo`.  Thinking that in the intro, `it` could be Fedora or Accumulo.